configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl

```diff
@@ -4,7 +4,7 @@ spec:
   containers:
   - image: ulissigroup/kubeflow:predictions
     imagePullPolicy: Always
-    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'4 GiB',--death-timeout, '600']
+    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'8 GiB',--death-timeout, '600']
```

Comment:
is it worth having a cpu relaxation worker? 4gb is plenty for other cpu workers if not doing inference

Reply:
https://github.com/ulissigroup/catlas/blob/main/configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl
Was the idea to request 9 and set the limit at 4 so workers would cut off if they started doing relaxations? If so we should probably make a new template for CPU relaxations